### DISCUSSION TITLE
Outreachy intern

### DISCUSSION THREAD

**Discussion by mildrette** [Open-Science-Community-Saudi-Arabia/Open-Science-Community-in-Saudi#20] [Outreachy 2022] [answered]
Hello!  everyone, I am N. Mildred Fonka an applicant for the outreachy internship and I will really love to work with you, learn from you and become part of the open science community.

**Comment by BatoolMM** [comment, marked as answer]
Very warm welcome @mildrette!
Thank you for joining, hope you have fun contributing to OSCSA!

**Reply by mildrette** [reply]
Yes i am thank you
